newjersey/navigator.business.nj.gov
# feat: [#188489128] non-profit question removal

## Description

Removed non-profit question from onboarding as it's something that can be filled out in the post onboarding legal structure selection part



### Ticket



This pull request resolves [#188489128](https://www.pivotaltracker.com/story/show/188489128).

### Approach



### Steps to Test

You should be able to, go through onboarding and see the question is gone

Got and select the non-profit legal structure and see that it still addes corresponding non-profit questions to roadmap

Trustees, charitable tax exempt, 



### Notes



## Code author checklist
